### ISSUE TITLE
returnImage not working on Android

### ISSUE THREAD

**Issue by nick990** [juliansteenbakker/mobile_scanner#289]
I'm using the package by importing the branch master:
```
dependencies:
  mobile_scanner:
    git:
      url: https://github.com/juliansteenbakker/mobile_scanner
      ref: master
```
I am getting an exception when setting `returnImage: true` on Android.
It seems to work fine on iOS.
```
  mobileScannerController = MobileScannerController(
      torchEnabled: true,
      returnImage: true,
    );
```
Exception:
```
E/flutter ( 8017): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: type 'Null' is not a subtype of type 'Uint8List' in type cast
E/flutter ( 8017): #0      MobileScannerController.handleEvent
package:mobile_scanner/src/mobile_scanner_controller.dart:103
E/flutter ( 8017): #1      new MobileScannerController.<anonymous closure>
package:mobile_scanner/src/mobile_scanner_controller.dart:90
E/flutter ( 8017): #2      _RootZone.runUnaryGuarded (dart:async/zone.dart:1586:10)
E/flutter ( 8017): #3      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:339:11)
E/flutter ( 8017): #4      _DelayedData.perform (dart:async/stream_impl.dart:515:14)
E/flutter ( 8017): #5      _PendingEvents.handleNext (dart:async/stream_impl.dart:620:11)
E/flutter ( 8017): #6      _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:591:7)
E/flutter ( 8017): #7      _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
E/flutter ( 8017): #8      _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
```

I need to know the dimension of the captured image from camera.
I think I understand this bug is fixed in #258 , but I can't test the feature because it has a lot of breaking changes.
With the published version on pub.dev, is there any way to do it ?

**Comment by ISL270** [comment]
I'm facing the same issue, is there a workaround?

**Comment by juliansteenbakker** [comment]
Hi, im aware of the issue and working on a fix for android.

**Comment by ISL270** [comment]
> Hi, im aware of the issue and working on a fix for android.

is it fixed yet?

**Comment by ISL270** [comment]
> Hi, im aware of the issue and working on a fix for android.

Can you guess approximately when this issue will be fixed?

**Comment by juliansteenbakker** [comment]
I will try to pick up some work this week and make a new release and the end of this week.

**Comment by nick990** [comment]
@juliansteenbakker Is there any news?

**Comment by juliansteenbakker** [comment]
Yes, i have released v3.0.0-beta.2 which fixes the image for both android and ios. Sorry for the long wait, it was quite a work.
